src/wp-includes/user.php

```diff
@@ -1004,9 +1004,8 @@ function get_blogs_of_user( $user_id, $all = false ) {
 
 	if ( ! empty( $site_ids ) ) {
 		$args = array(
-			'number'                 => '',
-			'site__in'               => $site_ids,
-			'update_site_meta_cache' => false,
```

Comment:
This will now queue the meta for lazy loading, whereas it wasn't loaded at all in the past.

Why is this change needed/an improvement?

Reply:
We made this change here https://github.com/WordPress/wordpress-develop/commit/82be24c885e4c69745ae0fdb14177617393a9317#diff-ff818ec19f0f94e4175f72bd2cc4cd67c455c0c50e5b651690a1e312c284ba06. 

Everywhere else WP_Site_Query a single site is loaded and site meta is not specially useful. This results an array of a users sites. So added site meta to the queue here, might have some database queries for some developers. We can do this in another change, but I wanted to have one place in core where use this new functionality. It helped me test it.